templates/hotgraphic.hbs

```diff
@@ -1,7 +1,7 @@
 {{! Maintainers - Kevin Corry}}
 <div class="hotgraphic-inner component-inner" role="region" aria-label="{{_globals._components._hotgraphic.ariaRegion}}" {{#if _globals._components._hotgraphic.ariaRegion}}tabindex="0"{{/if}}>
   {{> component this}}
-  <div class="hotgraphic-widget component-widget {{#if _useGraphicsAsPins}}tile{{else}}pin{{/if}}">
+  <div class="hotgraphic-widget component-widget {{#compare _pinStyle '=' 'tiles' }}tile{{else}}{{#if _useGraphicsAsPins}}tile{{else}}pins{{/if}}{{/compare}}">
```

Comment:
Suggest using the `equals` helper for readability. See [here](https://github.com/adaptlearning/adapt-contrib-languagePicker/blob/master/templates/languagePickerDrawerView.hbs#L4) for an example.